### PR TITLE
Fix EOL date

### PR DESCRIPTION
**Pull request by jkowall**
### Description
Fixing the EOL date for 1.x
 
### Issues Resolved
NA

### Check List
[X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by elfisher** [comment]
Thanks @jkowall for submitting this. This is my understanding too based on the maintenance policy [here](https://opensearch.org/releases.html).

**Review by elfisher — DISMISSED** [review]
lgtm. This looks accurate based on the published [maintenance policy](https://opensearch.org/releases.html).

**Comment by bbarani** [comment]
We should discuss more on this. Current policy states “ By default, versions will remain under maintenance until the next major version enters maintenance, or 1 year passes, whichever is longer”

**Review by bbarani — CHANGES_REQUESTED** [review]
This doesn’t reflect current stated policy.

File: releases.md [hunk 1]
```diff
@@ -65,7 +65,7 @@ The software maintainers will not back-port fixes or features to versions outsid
 
 | Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End |
 |:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:|
-| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   December 31, 2023    |
+| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   1 Year after 3.0 release date    |
```
Comment (dblock) [inline releases.md:68]: My OCD says let's capitalize consistently `1 Year after 3.0 Release Date` and format the table (align the `|`), but I'll shut up now.

**Comment by jkowall** [comment]
> This doesn’t reflect current stated policy. Refer to [#1801 (comment)](https://github.com/opensearch-project/project-website/pull/1801#issuecomment-1645738485). Based on the currently stated policy, 1.3.x will become EOL when 2.x goes in to maintenance mode.

It seems like there are two issues.  

1. When will 1.3.x enter maintenance?
2. When will 1.x enter maintenance?

I am trying to address number 2 here. Stating that 1.x will enter maintenance after the 3.0 release date, at which time EOL will be 1 year from then. This is also when 2.x becomes the maintenance release. 

We could redo the whole table to reflect this more clearly. Right now it states 2.0.0 is the minor version, which is wrong, we are at 2.9.0. I would suggest dropping the "minor version" column and instead focus on EOL of major versions.  

Up to you.

**Comment by bbarani** [comment]
@elfisher @jkowall Currently we actively support (and patch) and release only 1.3.x versions in 1.x line. Based on the current policy, 1.x will become EOL and 2.x will enter maintenance mode when 3.0 is released. As you have stated, we currently don't have 3.0 release date so we can state generically as  1.3.x EOL - > "GA release of 3.0". @dblock @CEHENKLE Please add your thoughts as well.

**Comment by dtaivpp** [comment]
>  1.3.x EOL - > "GA release of 3.0" 

I think this is the best path forward.

**Comment by elfisher** [comment]
> @elfisher @jkowall Currently we actively support (and patch) and release only 1.3.x versions in 1.x line. Based on the current policy, 1.x will become EOL and 2.x will enter maintenance mode when 3.0 is released. As you have stated, we currently don't have 3.0 release date so we can state generically as 1.3.x EOL - > "GA release of 3.0". @dblock @CEHENKLE Please add your thoughts as well.

I'm good with this proposal.

File: releases.md [hunk 2]
```diff
@@ -63,10 +63,10 @@ The software maintainers will not back-port fixes or features to versions outsid
 
 <div class="table-styler"></div>
 
-| Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End |
-|:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:|
-| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   December 31, 2023    |
-| 2             | 2.0.0                |   Current   |  May 26, 2022   |           N/A            |          N/A           |
+| Major Version | Latest Minor Version |   Status    | Initial Release | Maintenance Window Start | Maintenance Window End             |
+|:--------------|:---------------------|:-----------:|:---------------:|:------------------------:|:----------------------:            |
+| 1             | 1.3.x                | Maintenance |  July 12, 2021  |       May 26, 2022       |   1 Year after 3.0 Release Date    |
```
Comment (elfisher) [inline releases.md:68]: per the PR conversation let's change this to "GA release of 3.0." I can merge from there.

**Review by elfisher — CHANGES_REQUESTED** [review]
Minor wording adjustment per the PR conversation.

File: releases.md (hunk 2 above)
Comment (krisfreedain) [reply releases.md:68]: @jkowall - wanted to make sure you saw Eli's input - thanks

**Comment by dtaivpp** [comment]
I've made the requested changes and this is ready to go!

**Review by dtaivpp — APPROVED** [review]
Looks good!

**Pull request merged**